website/src/components/TutorialComponents/CodeBlock.tsx

```diff
@@ -126,6 +158,8 @@ CodeEditor.propTypes = {
     prism: PropTypes.object,
```

Comment:
perks of using typescript: use inline types in the function signature

Reply:
resolved in #98